exportModule.js

```diff
@@ -1,30 +1,29 @@
+const fs = require('fs');
 const data = require('./response.json');
 const loadData = require('./GetDataType.js');
 const states = require('./states.js');
-const stateLookup = require('./stateMap.js');
-function exportModule(data) {
-    const dataTypes = loadData();
-    const moduleJSON = {
-        "name": data.library.identifier.id,
-        "remarks": [],
-        "states": {
-            "Initial": new states.InitialState('Initial').toJSON(),
-            "Terminal": new states.TerminalState().toJSON()
-        }
-    };
-    dataTypes.forEach(function (temp, i) {
-        try {
-            const map = stateLookup[temp];
-            const stateName = `${temp.substring(21)}_${i}`;
-            const state = new map(stateName, [], {})
-            moduleJSON.states[stateName] = state.toJSON();
-        }
-        catch{
-            console.log('no corresponding state for that data type');
-            return;
-        }
-    })
-    return moduleJSON
+const factory = require('./factory.js');
+
+function exportModule() {
+  const dataTypes = loadData();
+  const moduleJSON = {
+    name: data.library.identifier.id,
+    remarks: [],
+    states: {
+      Initial: new states.InitialState('Initial').toJSON(),
+      Terminal: new states.TerminalState().toJSON(),
+    },
+  };
+  dataTypes.forEach(function (temp, i) {
```

Comment:
Looks like eslint wants you to use arrow functions here. You can replace it with the following and it will be the same behavior:

```suggestion
  dataTypes.forEach((temp, i) => {
```